qemu/tests/virtio_fs_share_data.py

```diff
@@ -292,15 +292,21 @@ def get_stdev(file):
                 session.cmd(cmd_autoreconf % fs_dest, 180)
                 session.cmd(cmd_configure.format(fs_dest), 180)
                 session.cmd(cmd_make % fs_dest, io_timeout)
-                session.cmd(cmd_pjdfstest % fs_dest, io_timeout)
+                status, output = session.cmd_status_output(cmd_pjdfstest % fs_dest, io_timeout)
```

Comment:
Hello @lkotek, it's a good habit to form a good code style, for this line, please limit the maximum length to 80 characters. Refer to: https://google.github.io/styleguide/pyguide.html#32-line-length

```suggestion
                s, o = session.cmd_status_output(cmd_pjdfstest % fs_dest,
                                                 io_timeout)
```
Or:
```suggestion
                status, output = session.cmd_status_output(
                    cmd_pjdfstest % fs_dest, io_timeout)
```

Reply:
Thanks I fully agree on that.  I committed your suggestion. Actually I wonder why `inspektor` didn't catch this before (I guess it was probably because one of the disabled PEPs).